### PR TITLE
fix: version should be 0 if not using smart contract for anchor

### PR DESCRIPTION
**Pull request by stephhuynh18**

### CODE REVIEW

**Review by oed — COMMENTED** [review]
Can we simply not include the `version` property if version is 0?

**Review by stbrody — APPROVED** [review]

File: src/services/anchor-service.ts [hunk 1]
```diff
@@ -330,14 +330,16 @@ export class AnchorService {
...
+    const ipfsAnchorProof = Object.assign(
+      {
+        blockNumber: tx.blockNumber,
+        blockTimestamp: tx.blockTimestamp,
+        root: merkleRootCid,
+        chainId: tx.chain,
+        txHash: txHashCid,
+      },
+      this.config.useSmartContractAnchors && { version: 1 }
+    )
```
Comment (oed) [inline src/services/anchor-service.ts:342]:
```suggestion
    const ipfsAnchorProof = {
      blockNumber: tx.blockNumber,
      blockTimestamp: tx.blockTimestamp,
      root: merkleRootCid,
      chainId: tx.chain,
      txHash: txHashCid,
    }
    if (this.config.useSmartContractAnchors) ipfsAnchorProof.version = 1
```
Can we make this a bit easier to read please?

**Review by oed — APPROVED** [review]
lgtm, one suggestion

File: src/services/anchor-service.ts (hunk 1 above)
Comment (oed) [reply src/services/anchor-service.ts:342]: Oh, typescript might not like that?

Comment (stephhuynh18) [reply src/services/anchor-service.ts]: only because the packages need to be updated to the latest version. I can work around

**Pull request merged**